tasks/python-reference-activity/README.md

```diff
@@ -0,0 +1,32 @@
+# @cumulus/python-reference-activity
+
+This 'task' is a reference activity implementation that is included with Cumulus to allow integration testing of the `Cumulus message adapter` and [`cumulus-process-py`](https://github.com/nasa/cumulus-process-py) with a deployed activity.
+
+## Development
+
+Developmental use of this lambda is intended to be simple - the processing activity runs an activity through the CMA and returns a static processing output, integration tests can be then built against the `Reference Activity` step in the `PythonProcess` workflow.
+
+### Requirements
+
+To develop against this task, you should be using python > 3.6 (CMA compatibility is baselined at 3.6).    Once you have a python env enabled:
+
+```bash
+pip install -r requirements.txt
+```
+
+### Build
+
+To update the container for a PR, you should run:
+
+```bash
+docker build -t cumuluss/cumulus-process-activity:{VERSION} .
+docker push cumuluss/cumulus-process-activity:{VERSION}
+```
+
+Then update [`python_reference_workflow.tf`](https://github.com/nasa/cumulus/blob/master/example/cumulus-tf/python_reference_workflow.tf) `python_processing_service` resource to utilize the correct image reference.
```

Comment:
```suggestion
Then update the  `python_processing_service` resource in [`python_reference_workflow.tf`](https://github.com/nasa/cumulus/blob/master/example/cumulus-tf/python_reference_workflow.tf) to utilize the correct image reference.
```

Reply:
Went with f416c8ed8 for consistency, then 77857f5ed